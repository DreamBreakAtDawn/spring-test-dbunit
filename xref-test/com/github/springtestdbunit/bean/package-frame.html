
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Spring Test DBUnit 1.3.0-SNAPSHOT Reference Package com.github.springtestdbunit.bean</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.springtestdbunit.bean</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="DatabaseConfigBeanTest.html" target="classFrame">DatabaseConfigBeanTest</a>
          	</li>
          	          	<li>
            	<a href="DatabaseDataSourceConnectionFactoryBeanTest.html" target="classFrame">DatabaseDataSourceConnectionFactoryBeanTest</a>
          	</li>
          	      	</ul>

	</body>
</html>