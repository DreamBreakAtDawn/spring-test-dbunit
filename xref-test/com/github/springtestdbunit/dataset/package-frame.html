
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Spring Test DBUnit 1.3.0-SNAPSHOT Reference Package com.github.springtestdbunit.dataset</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.springtestdbunit.dataset</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="FlatXmlDataSetLoaderTest.html" target="classFrame">FlatXmlDataSetLoaderTest</a>
          	</li>
          	          	<li>
            	<a href="ReplacementDataSetLoaderTest.html" target="classFrame">ReplacementDataSetLoaderTest</a>
          	</li>
          	          	<li>
            	<a href="XmlDataSetLoaderTest.html" target="classFrame">XmlDataSetLoaderTest</a>
          	</li>
          	      	</ul>

	</body>
</html>