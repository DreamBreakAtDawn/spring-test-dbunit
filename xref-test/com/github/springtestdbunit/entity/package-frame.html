
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Spring Test DBUnit 1.3.0-SNAPSHOT Reference Package com.github.springtestdbunit.entity</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.springtestdbunit.entity</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="EntityAssert.html" target="classFrame">EntityAssert</a>
          	</li>
          	          	<li>
            	<a href="HibernateImportTest.html" target="classFrame">HibernateImportTest</a>
          	</li>
          	          	<li>
            	<a href="OtherEntityAssert.html" target="classFrame">OtherEntityAssert</a>
          	</li>
          	          	<li>
            	<a href="OtherSampleEntity.html" target="classFrame">OtherSampleEntity</a>
          	</li>
          	          	<li>
            	<a href="SampleEntity.html" target="classFrame">SampleEntity</a>
          	</li>
          	      	</ul>

	</body>
</html>