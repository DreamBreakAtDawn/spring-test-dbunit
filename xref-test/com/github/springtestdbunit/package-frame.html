
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Spring Test DBUnit 1.3.0-SNAPSHOT Reference Package com.github.springtestdbunit</title>
		<link rel="stylesheet" type="text/css" href="../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.springtestdbunit</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">AbstractCustomDataSetLoader</a>
          	</li>
          	          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">CustomConfiguration</a>
          	</li>
          	          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">CustomDataSetLoader</a>
          	</li>
          	          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">CustomDatabaseOperationLookup</a>
          	</li>
          	          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">DbUnitTestExecutionListenerPrepareTest</a>
          	</li>
          	          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">EmptyDbUnitConfiguration</a>
          	</li>
          	          	<li>
            	<a href="TestExecutionListenerChainTest.html" target="classFrame">InvalidTestListener</a>
          	</li>
          	          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">LocalApplicationContextLoader</a>
          	</li>
          	          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">NoDbUnitConfiguration</a>
          	</li>
          	          	<li>
            	<a href="DbUnitTestExecutionListenerPrepareTest.html" target="classFrame">NonCreatableDataSetLoader</a>
          	</li>
          	          	<li>
            	<a href="TestExecutionListenerChainTest.html" target="classFrame">TestExecutionListenerChainTest</a>
          	</li>
          	          	<li>
            	<a href="TestExecutionListenerChainTest.html" target="classFrame">TestListener1</a>
          	</li>
          	          	<li>
            	<a href="TestExecutionListenerChainTest.html" target="classFrame">TestListener2</a>
          	</li>
          	          	<li>
            	<a href="TransactionDbUnitTestExecutionListenerTest.html" target="classFrame">TransactionDbUnitTestExecutionListenerTest</a>
          	</li>
          	      	</ul>

	</body>
</html>