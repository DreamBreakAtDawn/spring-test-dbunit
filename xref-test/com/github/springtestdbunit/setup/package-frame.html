
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Spring Test DBUnit 1.3.0-SNAPSHOT Reference Package com.github.springtestdbunit.setup</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.springtestdbunit.setup</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="CleanInsertSetupOnClassTest.html" target="classFrame">CleanInsertSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="CleanInsertSetupOnMethodTest.html" target="classFrame">CleanInsertSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteAllSetupOnClassTest.html" target="classFrame">DeleteAllSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteAllSetupOnMethodTest.html" target="classFrame">DeleteAllSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteSetupOnClassTest.html" target="classFrame">DeleteSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="DeleteSetupOnMethodTest.html" target="classFrame">DeleteSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="InsertSetupOnClassTest.html" target="classFrame">InsertSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="InsertSetupOnMethodTest.html" target="classFrame">InsertSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="MixedSetupOnClassAndMethodTest.html" target="classFrame">MixedSetupOnClassAndMethodTest</a>
          	</li>
          	          	<li>
            	<a href="MultipleInsertSetupOnClassTest.html" target="classFrame">MultipleInsertSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="RefereshSetupOnClassTest.html" target="classFrame">RefereshSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="RefereshSetupOnMethodTest.html" target="classFrame">RefereshSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="TruncateSetupOnClassTest.html" target="classFrame">TruncateSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="TruncateSetupOnMethodTest.html" target="classFrame">TruncateSetupOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="UpdateSetupOnClassTest.html" target="classFrame">UpdateSetupOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="UpdateSetupOnMethodTest.html" target="classFrame">UpdateSetupOnMethodTest</a>
          	</li>
          	      	</ul>

	</body>
</html>