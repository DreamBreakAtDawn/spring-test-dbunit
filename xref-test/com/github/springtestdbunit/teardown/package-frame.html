
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Spring Test DBUnit 1.3.0-SNAPSHOT Reference Package com.github.springtestdbunit.teardown</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.springtestdbunit.teardown</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="CleanInsertTearDownOnClass.html" target="classFrame">CleanInsertTearDownOnClass</a>
          	</li>
          	          	<li>
            	<a href="CleanInsertTearDownOnMethod.html" target="classFrame">CleanInsertTearDownOnMethod</a>
          	</li>
          	          	<li>
            	<a href="DeleteAllTearDownOnClass.html" target="classFrame">DeleteAllTearDownOnClass</a>
          	</li>
          	          	<li>
            	<a href="DeleteAllTearDownOnMethod.html" target="classFrame">DeleteAllTearDownOnMethod</a>
          	</li>
          	          	<li>
            	<a href="DeleteTearDownOnClass.html" target="classFrame">DeleteTearDownOnClass</a>
          	</li>
          	          	<li>
            	<a href="DeleteTearDownOnMethod.html" target="classFrame">DeleteTearDownOnMethod</a>
          	</li>
          	          	<li>
            	<a href="InsertTearDownOnClass.html" target="classFrame">InsertTearDownOnClass</a>
          	</li>
          	          	<li>
            	<a href="InsertTearDownOnMethod.html" target="classFrame">InsertTearDownOnMethod</a>
          	</li>
          	          	<li>
            	<a href="MixedTearDownOnClassAndMethodTest.html" target="classFrame">MixedTearDownOnClassAndMethodTest</a>
          	</li>
          	          	<li>
            	<a href="MultipleInsertTearDownOnClassTest.html" target="classFrame">MultipleInsertTearDownOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="RefreshTearDownOnClass.html" target="classFrame">RefreshTearDownOnClass</a>
          	</li>
          	          	<li>
            	<a href="RefreshTearDownOnMethod.html" target="classFrame">RefreshTearDownOnMethod</a>
          	</li>
          	          	<li>
            	<a href="TruncateTearDownOnClass.html" target="classFrame">TruncateTearDownOnClass</a>
          	</li>
          	          	<li>
            	<a href="TruncateTearDownOnMethod.html" target="classFrame">TruncateTearDownOnMethod</a>
          	</li>
          	          	<li>
            	<a href="UpdateTearDownOnClass.html" target="classFrame">UpdateTearDownOnClass</a>
          	</li>
          	          	<li>
            	<a href="UpdateTearDownOnMethod.html" target="classFrame">UpdateTearDownOnMethod</a>
          	</li>
          	      	</ul>

	</body>
</html>