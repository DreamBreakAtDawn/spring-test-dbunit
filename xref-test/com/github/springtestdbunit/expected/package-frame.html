
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xml:lang="en" lang="en">
	<head>
		<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
		<title>Spring Test DBUnit 1.3.0-SNAPSHOT Reference Package com.github.springtestdbunit.expected</title>
		<link rel="stylesheet" type="text/css" href="../../../../stylesheet.css" title="style" />
	</head>
	<body>

		<h3>
        	<a href="package-summary.html" target="classFrame">com.github.springtestdbunit.expected</a>
      	</h3>

      	<h3>Classes</h3>

      	<ul>
      		          	<li>
            	<a href="DoesNotSwallowExpectedFailureTest.html" target="classFrame">DoesNotSwallowExpectedFailureTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedFailureOnClassTest.html" target="classFrame">ExpectedFailureOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedFailureOnMethodTest.html" target="classFrame">ExpectedFailureOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedFailureOnMethodWithRepeatableReverseTest.html" target="classFrame">ExpectedFailureOnMethodWithRepeatableReverseTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedFailureOnMethodWithRepeatableTest.html" target="classFrame">ExpectedFailureOnMethodWithRepeatableTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictOnClassTest.html" target="classFrame">ExpectedNonStrictOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictOnMethodTest.html" target="classFrame">ExpectedNonStrictOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictUnorderedOnClassTest.html" target="classFrame">ExpectedNonStrictUnorderedOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictUnorderedOnMethodTest.html" target="classFrame">ExpectedNonStrictUnorderedOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictWithDtdTest.html" target="classFrame">ExpectedNonStrictWithDtdTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedOnClassAndMethodTest.html" target="classFrame">ExpectedOnClassAndMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedOnClassAndMethodWithoutOverrideTest.html" target="classFrame">ExpectedOnClassAndMethodWithoutOverrideTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryFailureOnMethodTest.html" target="classFrame">ExpectedQueryFailureOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryNonStrictFailureOnMethodTest.html" target="classFrame">ExpectedQueryNonStrictFailureOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryNonStrictOnMethodTest.html" target="classFrame">ExpectedQueryNonStrictOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryOnMethodTest.html" target="classFrame">ExpectedQueryOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">ExpectedQueryWithModifierOnMethodTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedTableNonStrictOnClassTest.html" target="classFrame">ExpectedTableNonStrictOnClassTest</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">InnerModifier</a>
          	</li>
          	          	<li>
            	<a href="ExpectedQueryWithModifierOnMethodTest.html" target="classFrame">OuterModifier</a>
          	</li>
          	          	<li>
            	<a href="ExpectedNonStrictWithDtdTest.html" target="classFrame">SampleEntityIdExclusionFilter</a>
          	</li>
          	      	</ul>

	</body>
</html>